test/testdata/deployednettemplates/recipes/fastnet/genesis.json

```diff
@@ -0,0 +1,104 @@
+{
+	"NetworkName": "",
+	"VersionModifier": "",
+	"ConsensusProtocol": "future",
+	"FirstPartKeyRound": 0,
+	"LastPartKeyRound": 30000,
+	"PartKeyDilution": 0,
+	"Wallets": [
+		{
+			"Name": "Wallet1",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet2",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet3",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet4",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet5",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet6",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet7",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet8",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet9",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet10",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet11",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet12",
+			"Stake": 4.166666666666667,
+			"Online": true
+		},
+		{
+			"Name": "Wallet13",
+			"Stake": 8.333333333333334,
+			"Online": false
+		},
+		{
+			"Name": "Wallet14",
+			"Stake": 8.333333333333334,
+			"Online": false
+		},
+		{
+			"Name": "Wallet15",
+			"Stake": 8.333333333333334,
+			"Online": false
+		},
+		{
+			"Name": "Wallet16",
+			"Stake": 8.333333333333334,
+			"Online": false
+		},
+		{
+			"Name": "Wallet17",
+			"Stake": 8.333333333333334,
+			"Online": false
+		},
+		{
+			"Name": "Wallet18",
+			"Stake": 8.333333333333334,
+			"Online": false
+		}
+	],
+	"FeeSink": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ",
```

Comment:
I think we want an actual feesink address here that is accessible, just in case.

Reply:
Added fee sink address.